hnclient-v2/src/main/java/ca/bc/gov/hlth/hnclientv2/authentication/ClientIdSecretBuilder.java

```diff
@@ -38,8 +41,10 @@ public ClientIdSecretBuilder(String clientId, String clientSecret) {
     }
 
     @Override
-    public ClientAuthentication build() {
-        logger.info("Building client ID and secret.");
+    public ClientAuthentication buildAuthenticationMethod() {
+        String methodName = LoggingUtil.getMethodName();
+
+        logger.info("{} - Using ClientId and Secret for Authentication",methodName);
```

Comment:
Made this logging consistent with our standard and updated the method name to be meaningful